Dockerfile

```diff
@@ -0,0 +1,31 @@
+# Copied from https://github.com/BretFisher/node-docker-good-defaults/blob/master/Dockerfile
+
+FROM node:8.9.3
+
+# Create app directory
+RUN mkdir -p /opt/app
+
+ARG NODE_ENV=production
+ENV NODE_ENV $NODE_ENV
+
+ARG PORT=80
+ENV PORT $PORT
+EXPOSE $PORT 1991
+
+ARG NPM_BIN=/opt/app/node_modules/.bin
+
+WORKDIR /opt
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY package*.json ./
+
+# If you are building your code for production
+# RUN npm install --only=production
+RUN npm install
+ENV PATH /opt/app/node_modules/.bin:$PATH
+
+WORKDIR /opt/app
+COPY . /opt/app
```

Comment:
These are different from the current edX docker standard of putting app code in `/edx/app/`. I think it would make sense to stick with that since this will be a cookie-cutter for all future IDAs.